go/vt/wrangler/reparent.go

```diff
@@ -405,6 +405,18 @@ func (wr *Wrangler) plannedReparentShardLocked(ctx context.Context, ev *events.R
 		return fmt.Errorf("old master tablet %v DemoteMaster failed: %v", topoproto.TabletAliasString(shardInfo.MasterAlias), err)
 	}
 
+	// When using RBR and semi-sync, new master can take a long time to find GTID set to send to replicas.
```

Comment:
I think this comment isn't quite accurate -- it's the combination of GTID and semi-sync that triggers the problem. RBR may make the binlogs larger relative to SBR, but I think we would have the same symptoms in SBR modes with large bulk inserts as well.